src/04-modality-specific-files/08-genetic-descriptor.md

```diff
@@ -0,0 +1,60 @@
+  "License": "CC0",  
+  "Authors": ["1st author", "2nd author"],  
+  "Funding": "list your funding sources",  
+  "GeneticDataBase": ["Name", "www",],  
+  "GeneticDescriptor": ["data paper", "10.0.2.3/dfjj.10"]  
+}
+
+## sub- and participant.tsv
+
+One issue with having two repositories (one for imaging, one for genetics) is naming of subjects. A simple solution is to use the same ID in both datasets. Alternatively, one can encode the name used in the genetic database in the participant.tsv file. The column `GeneticID` (OPTIONAL) encodes the ID used for the same subject in the `GeneticDataBase` referred to in the dataset_description.json file.
+
+Information about the presence/absence of specific genetic markers can be and should be shared simply in the participants.tsv file adding optopnal columns.
+
+`participants.tsv` example:
+
+```Text
+participant_id  age sex group GeneticID	IDH Mutation
+sub-control01 34  M control 124587	yes
+sub-control02 12  F control 548936	yes
+sub-patient01 33  F patient 489634	no
+```
+
+## genetic_info.json
+
+This file is the descriptor of the genetic information available either in the participant tsv file and/or the genetic database described in the dataset_description.json. The 'GeneticLevel' and 'SampleOrigin' are the only two mandatory fields.
+
+| Field name   | Definition | Values |
+| :----------- | :--------- | :------|
+| GeneticLevel | MANDATORY Describes the level of analysis | `Genetic`, `Genomic`, `Epigenomic`, `Transcriptomic`, `Metabolomic`, or `Proteomic` |
+| AnalyticalApproach | OPTIONAL Methodology used to analyse the GeneticLevel | Value must be taken from [gapsolr](https://www.ncbi.nlm.nih.gov/projects/gapsolr/facets.html) under /Study/Molecular Data Type, for instance `SNP Genotypes (Array)` or `Methylation (CpG)` |
+| SampleOrigin | MANDATORY Describes from which tissue the genetic information was extracted from | `blood`, `saliva`, `brain`, `csf`, `breast milk`, `bile`, `amniotic fluid`, `other biospecimen` |
+| TissueOrigin | OPTIONAL Describes the type of tissue analyzed for SampleOrigin `brain` | `gray matter`, `white matter`, `csf`, `meninges`, `macrovascular` or `microvascular` |
+| BrainLocation | OPTIONAL Refers to the location in space of the TissueOrigin | `MNI coordinate` or a `label` taken from the [Allen Brain Atlas](http://atlas.brain-map.org/atlas?atlas=265297125#atlas=265297125&plate=112360888&structure=4392&x=40348.15104166667&y=46928.75&zoom=-7&resolution=206.60&z=3) possibly `layer` to refer to layer-specific gene expression, which can also tie up with laminar fMRI |
+| CellType | OPTIONAL Describes the type of cell analyzed | Value should come from the [cell ontology](http://obofoundry.org/ontology/cl.html) |
+
+`genetic_info.json` example:
+
+{  
+  "GeneticLevel": "Genetic",  
+  "AnalyticalApproach": "SNP Genotypes",  
+  "SampleOrigin": "brain",  
+  "TissueOrigin": "gray matter",  
+  "CellType":  "neuron",  
+  "BrainLocation": "[-30 -15 10]"  
+}
```

Comment:
again, please format this as a JSON codeblock, as described in my comment above.

Reply:
`genetic_info.json` example:
```JSON   
{  
  "GeneticLevel": "Genetic",
  "AnalyticalApproach": "SNP Genotypes", "SampleOrigin": "brain",
  "TissueOrigin": "gray matter",
  "CellType":  "neuron",
  "BrainLocation": "[-30 -15 10]"
}
```